Menus/MainMenu/MainMenu.cs

```diff
@@ -273,14 +277,9 @@ void OnButtonHovered(Transform rayOrigin)
 			this.Pulse(this.RequestNodeFromRayOrigin(rayOrigin), m_ButtonHoverPulse);
 		}
 
-		void OnOpening()
-		{
-			this.Pulse(node, m_ShowPulse);
-		}
-
-		void OnClosing()
+		public void SendVisibilityPulse()
```

Comment:
Yeah. The issue is that we need to distinguish between a temp-hidden and perma-hidden state because we specifically needed to cut haptics on auto-hide.

I could just add a "withHaptics" argument or perhaps "isTemporary" if we want to go with that semantic.